store/sqlstore/upgrade.go

```diff
@@ -969,6 +969,8 @@ func upgradeDatabaseToVersion533(sqlStore *SqlStore) {
 
 func upgradeDatabaseToVersion534(sqlStore *SqlStore) {
 	// if shouldPerformUpgrade(sqlStore, Version5330, Version5340) {
+	sqlStore.CreateColumnIfNotExists("ChannelMembers", "MentionCountRoot", "bigint", "bigint", "0")
```

Comment:
Why are you adding the default value?, that can generate some problems with the execution of the migration, adding a NULL column should work fine, adding a column with a default value, I guess it can generate problems. Can we confirm that?

Probably this is also generating the discrepancy between the migration and the fresh schema.

Another thing. Can we use the new migrations system? // @nronas

Reply:
@jespino - I don't think having the default value as NULL is a good idea... @streamer45 - can you weigh in on ALTER with default vs ALTER without a default value in terms of performance?

Reply:
@jespino - I'll move to NULL be default (will help with lazy migration anyways)